setup.py

```diff
@@ -27,11 +27,11 @@
     url=__url__,
     packages=find_packages(where="src"),
     package_dir={"": "src"},
-    package_data=dict(hyperlink=["py.typed",],),
+    package_data=dict(hyperlink=["py.typed"]),
     zip_safe=False,
     license=__license__,
     platforms="any",
-    install_requires=["idna>=2.5", 'typing ; python_version<"3.5"',],
+    install_requires=["idna>=2.5", 'typing ; python_version<"3.5"'],
```

Comment:
Because we're adding `setup.py` to the files Flake8 looks at.

Reply:
Appreciate the explanation :)